src/ImgixTag.js

```diff
@@ -81,7 +83,7 @@ var ImgixTag = (function() {
 
     if (this.settings.includeLibraryParam) {
       params.ixlib = 'imgixjs-' + imgix.VERSION;
-    }
+		}
```

Comment:
[nit] Tabs & spaces, again. Is this perchance a Prettify thing that's breaking down?

Reply:
Gah! I think it's time to tame the beast and work out what's going on here.